### PR TITLE
Adding Living Style Guide (as part of Rails project)

### PR DESCRIPTION
**Pull request by myabc**
https://www.openproject.org/work_packages/7630

I'm not a fan of monolithic applications, but for now at least, it's a lot less complex to keep the style guide in the same repository and use the built-in LSG Rails integration. Replaces #1280.

Style guide is available at `localhost:3000/assets/styleguide.html`

### CODE REVIEW

**Comment by tilt** [comment]
heroic

File: Gemfile [hunk 1]
```diff
@@ -107,11 +107,14 @@ group :production do
 end
 
 group :assets do
-  gem 'sass-rails',   '~> 3.2.3'
+  gem 'sass-rails', git: 'https://github.com/guilleiguaran/sass-rails.git', branch: 'backport'
```
Comment (tessi) [inline Gemfile]: I'd love to see a comment on why we have to use he git repo (instead of a version from rubygems here)

Comment (myabc) [inline Gemfile]: @tessi in addition to the commit message? (c0dc50e9). Our `Gemfile` is already rather cluttered IMO.

I _do_ have reservations about depending on relatively _obscure_ forks. Perhaps a better strategy for us would be to fork this fork into the finnlabs account?

These niggling asset problems won't go away until we upgrade to Rails 4.0.

Comment (tessi) [inline Gemfile]: Ah, haven't seen the commit message. I'm fine with that :)

**Comment by myabc** [comment]
I dislike the hack in b0f758fc, but otherwise this should be mergeable.

**Pull request merged**